its_live_monitoring/src/main.py

```diff
@@ -48,13 +91,52 @@ def deduplicate_hyp3_pairs(pairs: gpd.GeoDataFrame) -> gpd.GeoDataFrame:
     Returns:
          The pairs GeoDataFrame with any already submitted pairs removed.
     """
-    jobs = HYP3.find_jobs(
+    corners_of_pairs = [list(geom.exterior.coords)[:-1] for geom in pairs['geometry']]
+    regions_for_pairs = [{point_to_region(lat, lon) for lon, lat in corners} for corners in corners_of_pairs]
+
+    s2_prefix = 'velocity_image_pair/sentinel2/v02/'
+    landsat_prefix = 'velocity_image_pair/landsatOLI/v02/'
+    prefix = s2_prefix if pairs['reference'][0].startswith('S2') else landsat_prefix
+    tile_prefixes = [{prefix + region for region in regions} for regions in regions_for_pairs]
+    pairs['tile_prefixes'] = tile_prefixes
```

Comment:
```suggestion
 
    pairs['tile_prefixes'] = [{prefix + region for region in regions} for regions in regions_for_pairs]
```
I think a whitespace line would be sufficient to let this be a single line